set-version.sh

```diff
@@ -126,6 +126,56 @@ function update_all_pom_version_tags {
 
 
 
+    # cat $source_file | sed "0,/<version>.*<\/version>$/<version>$component_version<\/version>/}" > $temp_file
+    # rc=$?; if [[ "${rc}" != "0" ]]; then error "Failed to set version into $source_file file."; exit 1; fi
```

Comment:
Do we need this code? Seems like we've got it commented in a few places here so might be good to remove if not

Reply:
Good spot, I'll remove these